apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -77,6 +81,8 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
         // Must be <= supportRequiredPct to avoid votes being impossible to pass
         uint64 minAcceptQuorumPct;
 
+        // Note: we could re-work these concepts to be against the start of the vote rather than the end
+        //       E.g. overrule window is more like "allowed time for representatives to vote"
```

Comment:
We discussed this briefly offline, but I think we should give a go at this "refactor" to see if it can simplify the code and our mental model of the timings.

The best description I have for these values' behaviour is "the amount of voting-eligible time before/after which X happens", where X is one of:

- Time allowed for representatives to vote on others' behalf
- Time to take the initial quiet ending snapshot

----

Flipping it like this gets a bit harder to conceptualize for those who don't want those settings (e.g. "I don't want quiet ending" --> "your snapshot time should be the same as your vote time" rather than 0), but I think it's worth it.

Reply:
Addressed in 8765ccf206f6d4e8ff78e51d5e84d9dd7a77810d